test/api/postman_collection.json

```diff
@@ -11974,7 +13508,7 @@
 								"method": "GET",
 								"header": [],
 								"url": {
-									"raw": "{{baseUrl}}/stigs/ccis/:cci?projection=stigs&projection=references&projection=emassAp",
+									"raw": "{{baseUrl}}/stigs/ccis/:cci",
```

Comment:
I can see that the projections removed from here are not tested for, but this modification seems to have no nexus with the issue?